lib/everypolitician/popolo/event.rb

```diff
@@ -38,8 +38,29 @@ def wikidata
       attr_reader :document
     end
 
+    class DynamicEventClassFinder
```

Comment:
Is there a better name for this, based on what it _is_, rather than what it _does_?

(Also, what's the path to adding more of these, e.g. for having separate `Party` and `Legislature` subclasses of `Organization`? Do we extend _this_? Or add a parallel version somewhere?)